dbt_subprojects/hourly_spellbook/models/_sector/lending/borrow/sei/_schema.yml

```diff
@@ -0,0 +1,124 @@
+      sector: lending
+      project: yei
+      contributors: hosuke
+    config:
+      tags: ['lending', 'borrow', 'yei', 'compound', 'sei']
+    description: "All lending borrow transactions on sei"
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - transaction_type
+            - token_address
+            - tx_hash
+            - evt_index
+    columns:
+      - &blockchain
+        name: blockchain
+        description: "Blockchain name"
+        tests:
+          - not_null
+      - &project
+        name: project
+        description: "Project name"
+        tests:
+          - not_null
+      - &version
+        name: version
+        description: "Version of the contract built and deployed by the lending project"
+        tests:
+          - not_null
+      - &transaction_type
+        name: transaction_type
+        description: "Transaction type"
+        tests:
+          - not_null
+      - &token_address
+        name: token_address
+        description: "Token contract address"
+        tests:
+          - not_null
+      - &borrower
+        name: borrower
+        description: "Borrower wallet address"
+      - &on_behalf_of
+        name: on_behalf_of
+        description: "Wallet address tx was executed on behalf of"
+      - &repayer
+        name: repayer
+        description: "Repayer wallet address"
+      - &liquidator
+        name: liquidator
+        description: "Liquidator wallet address"
+      - &amount
+        name: amount
+        description: "Token amount present in the transaction"
+      - &block_month
+        name: block_month
+        description: "Block month column used to partition data in this table"
+      - &block_time
+        name: block_time
+        description: "Timestamp for block event time in UTC"
+        tests:
+          - not_null
+      - &block_number
+        name: block_number
+        description: "Event block number"
+        tests:
+          - not_null
+      - &project_contract_address
+        name: project_contract_address
+        description: "Project contract address"
+        tests:
+          - not_null
+      - &tx_hash
+        name: tx_hash
+        description: "Transaction hash of the event"
+        tests:
+          - not_null
+      - &evt_index
+        name: evt_index
+        description: "Event index"
+        tests:
+          - not_null
+
+  - name: yei_sei_base_borrow
+    meta:
+      blockchain: sei
+      sector: lending
+      project: yei
+      contributors: hosuke
+    config:
+      tags: ['lending', 'borrow', 'aave', 'sei', 'yei']
+    description: "YEI borrow transactions on sei"
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - transaction_type
+            - token_address
+            - tx_hash
+            - evt_index
+#      - check_lending_base_borrow_seed:
+#          seed_file: ref('yei_sei_base_borrow_seed')
```

Comment:
The reason I do not include seed and tests in this PR is because yei_sei_base_[borrow|supplay|flashloans] do not have any records at the moment.

This PR serves as a skeleton for lending sector of sei chain.